tumblr-photo-video-ripper.py

```diff
@@ -85,10 +88,42 @@ def _download_medium(medium_type, medium_url, folder_name):
                                                         medium_url))
 
 
+def usage():
+    print 'Please create file sites.txt under this same directory'
```

Comment:
This syntax works only on Python2. It will raise exceptions on Python3. Please fix these. Thanks so much for your contribution.